### DISCUSSION TITLE
latest mastercontainer docker image can't start "domaincheck"

### DISCUSSION THREAD

**Discussion by Tab3r** [nextcloud/all-in-one#3357] [Questions] [answered]
Hi, i'm trying to remount a Nextcloud AOI from scratch, in a Terramaster NAS, and i found a strange error, only in the Nextcloud AOI (we have several services running without problem).

I run "mastercontainer" without problem, give me the master pass, enter the pass, and then an error raised in the web:

![image](https://github.com/nextcloud/all-in-one/assets/784833/e22f41c2-23be-4a37-b5e6-651f1992aa1a)

Looking in the log from "mastercontainer" i found: 

```
NOTICE: PHP message: Could not start domaincheck container: Server error: `POST http://localhost/v1.41/containers/nextcloud-aio-domaincheck/start` resulted in a `500 Internal Server Error` response:
exec: "docker-init": executable file not found in $PATH
```
If I try to run the container created by de "mastercontainer" manually from the console, same error... the container not start with the "exec" error.

But If a get a console for "mastercontainer" I can create a container with the docker console: `"docker run nextcloud/aio-domaincheck:latest ..."` and docker doesn't show errors...

Is a "mastercontainer" problem or docker problem?

Same problem is asked in the Terramaster support forum: [Link](https://forum.terra-master.com/en/viewtopic.php?f=43&t=5175)

### Steps to reproduce
1. Create a container for "mastercontainer"
2. Continue to domain check container creation
3. Get error.

### Expected behavior 
Enter in the regular container and configuration for AOI.

### Actual behavior 
Blocked in a error, container for domaincheck is created, but can't start with error:
`exec: "docker-init": executable file not found in $PATH` 

### Host OS 
Terramaster OS, Linux, amd64

#### Nextcloud AIO version 
Nextcloud AIO v7.2.1

#### Current channel 
stable

#### Other valuable info 
Info about the OS, and docker version:

```
[david@TNAS-F2-223 ~]# docker version
Client:
 Version:           20.10.17
 API version:       1.41
 Go version:        go1.18.6
 Git commit:        20.10.17
 Built:             unknown-buildtime
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.17
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.6
  Git commit:       buildroot
  Built:            
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.6
  GitCommit:        
 runc:
  Version:          1.1.3
  GitCommit:        
[david@TNAS-F2-223 ~]# uname -r
5.15.59
```

**Comment by Tab3r** [comment]
I downgraded to "nextcloud/all-in-one:20230817_065941-latest" and it works!...

The problem isn't Terramaster itself, some changes in the last "nextcloud/all-in-one" doesn't work...

I can do more testing, some ideas?

Thank you for your time!!!

**Comment by szaimen** [comment, marked as answer]
Hi, the problem needs to be fixed in Terramaster as they dont seem to support the init flag correctly. (It is a default docker thing so rather nothing we can fix in AIO.)
